config-solaris.md

```diff
@@ -90,7 +90,7 @@ Mapped to `allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
 * **`configureAllowedAddress`** *(string, OPTIONAL)* If configureAllowedAddress is set to true, the addresses specified by allowedAddress are automatically configured on the interface each time the container starts.
 When it is set to false, the allowedAddress will not be configured on container start.
 Mapped to `configure-allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
-* **`defrouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
+* **`defaultRouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
```

Comment:
I take it back, simply on account of it being `def` in the Solaris documentation. Reverted that change.